### ISSUE TITLE
Can't enable openebs Addon.

### ISSUE THREAD

**Issue by britnoth** [ubuntu/microk8s#2611]
I can't enable the openebs addon on MicroK8s.

I have the following installed:

- MicroK8s version 1.21/stable
- It is running on a Ubuntu Server, version 20.04.3. There are 3 Ubuntu servers in an HA cluster. 
- iscsid is running ok.

I have successfully enabled the following addons:

> enabled:
    dashboard            # The Kubernetes dashboard
    dns                  # CoreDNS
    fluentd              # Elasticsearch-Fluentd-Kibana logging and monitoring
    ha-cluster           # Configure high availability on the current node
    helm3                # Helm 3 - Kubernetes package manager
    ingress              # Ingress controller for external access
    metrics-server       # K8s Metrics Server for API access to service metrics
    prometheus           # Prometheus operator for monitoring and logging
    rbac                 # Role-Based Access Control for authorisation

When I try:

```microk8s enable openebs```

I get:

>Addon dns is already enabled.
Addon helm3 is already enabled.
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /var/snap/microk8s/2487/credentials/client.config
"openebs" already exists with the same configuration, skipping
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /var/snap/microk8s/2487/credentials/client.config
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "openebs" chart repository
Update Complete. ⎈Happy Helming!⎈
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /var/snap/microk8s/2487/credentials/client.config
Error: cannot re-use a name that is still in use

This seems to be a problem with the following helm3 install command in /snap/microk8s/current/actions/enable.openebs.sh:

```
$HELM -n openebs install openebs openebs/openebs \
    --set varDirectoryPath.baseDir="$SNAP_COMMON/var/openebs" \
    --set jiva.defaultStoragePath="$SNAP_COMMON/var/openebs" \
    --set localprovisioner.basePath="$SNAP_COMMON/var/openebs/local" \
    --set ndm.sparse.path="$SNAP_COMMON/var/openebs/sparse"
```

I have tried setting the version of openebs to --version '2.x.x' as suggested by OhMyMndy in #2600, but the result is the same.

What am I doing wrong or is this addon broken?

[inspection-report-20210928_170818.tar.gz](https://github.com/ubuntu/microk8s/files/7245850/inspection-report-20210928_170818.tar.gz)

**Comment by ktsakalozos** [comment]
Hi @britnoth,

Yes this addon is currently broken. The fix submitted by the openebs team is making its way through our release process. At the moment the 1.21/edge should be patched. Could you refresh to that channel and retry enabling the addon? `sudo snap refresh microk8s --channel=1.21/edge`. Apologies for the inconvenience.

**Comment by britnoth** [comment]
Thanks ktsakalozos.

I refreshed MicroK8s with version 1.21/edge, as you suggested.

Unfortunately, when I tried ```microk8s enable openebs``` it still failed with:

> Error: cannot re-use a name that is still in use

I had to modify my copy of the microk8s addon installation script (enable.openebs.sh) to get helm3 to work with openebs:

* Changed: ```$HELM -n openebs install```
* To: ```$HELM -n openebs upgrade --install```

I also (possibly unnecessarily) specified the openebs version:

```--version '2.11.0'```

openebs then successfully installed.

I have only just started to use helm but I think my error was caused by a previous failed install of openebs. If I now check helm's history  using:

```microk8s helm3 -n openebs history openebs```

Then I get:

> REVISION        UPDATED                         STATUS          CHART           APP VERSION DESCRIPTION
1               Tue Sep 28 11:39:43 2021        superseded      openebs-3.0.0   3.0.0       Install complete
2               Wed Sep 29 09:07:54 2021        deployed        openebs-2.11.0  2.11.0      Upgrade complete

but a version 3.0.0 openebs which worked with MicroK8s was never actually installed.
